docs/ref/forms/fields.txt

```diff
@@ -1215,6 +1215,9 @@ method::
         field will be derived, and which will be used to validate the
         user's selection.
 
+    ``ModelMultipleChoiceField`` also takes one optional argument,
```

Comment:
It doesn't appear to be an optional argument for this subclass. When a ModelMultipleChoiceField initializes the parent ModelChoiceField it [forces the empty_label arg to None](https://github.com/django/django/blob/5ec64f96b2d83ec3c0ef574f52e4767a440017b8/django/forms/models.py#L1243).

So attempting to set a value causes a TypeError due to multiple values for empty_label.
`TypeError: __init__() got multiple values for argument 'empty_label'`